R/rownames.R

```diff
@@ -65,6 +65,31 @@ rownames_to_column <- function(df, var = "rowname") {
   new_df
 }
 
```

Comment:
The function needs to be documented and exported. I think documentation can go to the same page as that for `rownames_to_column()`.